### DISCUSSION TITLE
Changing email-confirmation URL var using a custom one

### DISCUSSION THREAD

**Discussion by gino8080** [strapi/strapi#7311] [Legacy Topics]
I could not found an answer after searching around on issues:

As subject, I want to change the <%= URL %> variable in the confirmation email template

I want to send a confirmation email link that points to a React Client on another domain
But i don't want to hardcode it in the template email, I was thinking to something like a

<%= MY_CLIENT_URL %> custom variable to be used in the template, 

so i can use an ENV in different environments, like for example:
localhost:3000 in development
and 
myStrapiReactClient.com on production

any suggestion?

thank you

**Comment by derrickmehaffy** [comment]
Hello!

In this particular case if you navigate to the advanced settings in the Users & Permissions plugin (such as: http://localhost:1337/admin/plugins/users-permissions/advanced ) You will see the option that sets this URL:

![image](https://user-images.githubusercontent.com/8593673/89341879-d195ac00-d656-11ea-92c2-cbd4a8b3e966.png)

This data is saved to the database, so as long as you are using a different database for each environment this should change based on that value. However if you wish to change this logic you can use the [extensions system](https://strapi.io/documentation/v3.x/getting-started/troubleshooting.html#how-do-i-customize-a-plugin) and update the following code that handles this variable: https://github.com/strapi/strapi/blob/a8bb45da52a0f7e8e3a47c07f306a3662d8e0cc1/packages/strapi-plugin-users-permissions/controllers/Auth.js#L495-L506

**Reply by gino8080** [reply]
great thank you very much
